client/src/main/java/io/pravega/client/segment/impl/EventSegmentReaderImpl.java

```diff
@@ -98,6 +98,7 @@ private ByteBuffer readEvent(long firstByteTimeoutMillis) throws EndOfSegmentExc
         headerReadingBuffer.clear();
         int read = in.read(headerReadingBuffer, firstByteTimeoutMillis);
         if (read == 0) {
+            log.debug("No of bytes read is empty for segment id {}.", in.getSegmentId());
```

Comment:
This message is confusing. Please, rephrase to something like: `Empty read for segment id {}` or `No bytes read for segment id {}`. Log messages are important for debugging, so please make an effort to define clear and concise messages.

Reply:
Done as suggested.